hops/experiment_impl/util/experiment_utils.py

```diff
@@ -473,12 +473,16 @@ def _attach_experiment_xattr(ml_id, json_data, op_type):
 
     response = util.send_request('PUT', resource_url, data=json_data, headers=headers)
 
-    response_object = response.json()
+    try:
+        response_object = response.json()
+    except ValueError:
+        response_object = {}
```

Comment:
When is this ValueError raised? Is the backend sending something that's not a json?

Reply:
Yes, if we send a wrong input it will fail on unmarshaling and payara will send a string error